docs/extensions/extension_sandboxing.md

```diff
@@ -56,6 +56,15 @@ permission is a superset of `read` that allows for execution. While `read`,
 `write`, and `run` accept either a path to be allowed or a boolean, `net` only
 allows for a boolean value.
 
+## Limitations
+
+The `PhylumApi.runSandboxed` method is the only allowed means of spawning
+child processes from an extension. In order to prevent extension to escape the
+sandbox, methods such as `Deno.run` are explicitly disabled.
```

Comment:
"Methods such as" is incredibly unclear here. Makes it sound like we deny a bunch of other stuff but don't actually document it.

Should just be "the `Deno.run` is explicitly disabled." or something like that. Sounds a little nicer if you turn it around and start with `The Deno.run method is explicitly disabled, to prevent bla bla bla`.

Reply:
Wanted to discuss this -- in newer releases of Deno, I've noticed there are other methods of spawning processes besides `run` such as [`spawnChild`](https://doc.deno.land/deno/unstable/~/Deno.spawnChild). We don't necessarily want to bump Deno releases anytime soon, and I don't think those methods exist for the runtime version we are using; but if we ever do, we should be aware of those, otherwise escaping the sandbox could be a possibility.

I'm cool with simplifying the wording for the time being.